src/utils/bridge/sub/classes/history.ts

```diff
@@ -336,57 +369,76 @@ class SubBridgeHistory extends SubNetworksConnector {
       soraEvents.slice(requestStatusUpdateEventIndex),
       this.soraApi
     );
-    const soraParachainBlockId = history.parachainBlockId as string;
-    // SORA Parachain block events
-    const soraParachainEvents = await api.system.getBlockEvents(soraParachainBlockId, this.parachainApi);
-    // SORA Parachain message sended to SORA
-    const messageToSoraEvent = soraParachainEvents.find((e) => {
-      if (!this.parachainApi.events.substrateBridgeOutboundChannel.MessageAccepted.is(e.event)) return false;
-
-      const [soraParachainBatchNonce, soraParachainMessageNonce] = getMessageAcceptedNonces([e], this.parachainApi);
-
-      return soraParachainBatchNonce === soraBatchNonce && soraParachainMessageNonce === soraMessageNonce;
+    // api for Standalone network or SORA parachain
+    const networkApi = subBridgeApi.isStandalone(history.externalNetwork!) ? this.externalApi : this.parachainApi!;
+    const networkBlockId = history.externalBlockId as string;
+    // Network block events
+    const networkEvents = await api.system.getBlockEvents(networkBlockId, networkApi);
+    // Network message sended to SORA
+    const messageToSoraEvent = networkEvents.find((e) => {
+      if (!networkApi.events.substrateBridgeOutboundChannel.MessageAccepted.is(e.event)) return false;
+
+      const [networkBatchNonce, networkMessageNonce] = getMessageAcceptedNonces([e], networkApi);
+
+      return networkBatchNonce === soraBatchNonce && networkMessageNonce === soraMessageNonce;
     });
 
     if (!messageToSoraEvent) {
-      throw new Error(`Message sended to SORA from SORA Parachain block "${history.parachainBlockId}" not found`);
+      throw new Error(
+        `[${history.id}] Message sended to SORA from external network block "${networkBlockId}" not found`
+      );
     }
 
-    const soraParachainExtrinsicIndex = messageToSoraEvent.phase.asApplyExtrinsic.toNumber();
-    const soraParachainExtrinsicEvents = getTxEvents(soraParachainEvents, soraParachainExtrinsicIndex);
+    const networkExtrinsicIndex = messageToSoraEvent.phase.asApplyExtrinsic.toNumber();
+    const networkExtrinsicEvents = getTxEvents(networkEvents, networkExtrinsicIndex);
+
+    if (history.externalNetwork! === SubNetworkId.Liberland) {
```

Comment:
Confusing combinations of non-null assertion and equal test like "a! == b", which looks very similar to not equal "a !== b". Please try to avoid `!` here